editor.planx.uk/src/pages/FlowEditor/components/PreviewBrowser.tsx

```diff
@@ -17,28 +18,22 @@ import { TYPES } from "../../../@planx/components/types";
 import Questions from "../../Preview/Questions";
 import { useStore } from "../lib/store";
 
-const useStyles = makeStyles((theme) => ({
-  console: {
-    overflow: "auto",
-    padding: 20,
-    maxHeight: "50%",
-  },
-  previewContainer: {
-    overflow: "auto",
-    flex: 1,
-    background: "#fff",
-    padding: theme.spacing(3),
-  },
-  refreshButton: {
-    color: "inherit",
-  },
-  header: {
-    display: "flex",
-    flexDirection: "column",
-  },
-  publishButton: {
-    width: "100%",
-  },
+const Console = styled(Box)(() => ({
+  overflow: "auto",
+  padding: 20,
+  maxHeight: "50%",
+}));
+
+const PreviewContainer = styled(Box)(({ theme }) => ({
+  overflow: "auto",
+  flex: 1,
+  background: "#fff",
+  paddingTop: theme.spacing(3),
```

Comment:
I think this change from `padding` above to only `paddingTop` here means that content is squished in the sidebar now: 
![Screenshot from 2023-06-16 09-05-56](https://github.com/theopensystemslab/planx-new/assets/5132349/f0e37f98-ede7-4293-aae5-6d307a44b7bb)

Reply:
Good catch thank you - this came from a merge conflict and I missed it.